lib/rubocop/cop/rails/timeout_from_transaction.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+require 'rubocop'
+
+module RuboCop
+  module Cop
+    module Rails
+      # This cop checks for transactions wrapped in timeouts,
+      # motivated by change in Rails behavior that will
+      # cause exited transactions to be rolled back
+      # rather than committed (pre ActiveRecord 7 behavior).
```

Comment:
```suggestion
      # rather than committed (pre Active Record 7.0 behavior).
```

> The proper names of [Rails](http://d.hatena.ne.jp/keyword/Rails) components have a space in between the words, like "Active Support". `ActiveRecord` is a [Ruby](http://d.hatena.ne.jp/keyword/Ruby) module, whereas Active Record is an ORM.

https://guides.rubyonrails.org/api_documentation_guidelines.html#wording